packages/astroport-governance/src/assembly.rs

```diff
@@ -251,7 +255,7 @@ impl Proposal {
         if !self
             .title
             .chars()
-            .all(|c| c.is_alphanumeric() || c.is_ascii_whitespace() || ALLOWED_SIGNS.contains(&c))
+            .all(|c| c.is_alphanumeric() || c.is_ascii_whitespace() || ALLOWED_SIGNS.contains(c))
```

Comment:
This check can be converted to regexp matching as well.
Regexp: ```const TITLE_REGEX: &str = r#"[\w\s!&#()*+'"-./]"#```

Reply:
Ok, changed, thanks